### ISSUE TITLE
It says i dont have enough storage

### ISSUE THREAD

**Issue by skuxkid123** [Botspot/wor-flasher#47]

**Comment by cycool29** [comment]
`DL_DIR` variable is the directory you want to download the installation components before flashing you SD card, and it defaults to ` ~/wor-flasher-files`.

This error occurs as you don't have sufficient space to download the installation components in your system drive.
The solution is plug another storage and change the `DL_DIR` variable to the storage to store the files. 

To change `DL_DIR` variable:

## `GUI`

At the `Installation Overview` page, click `Advanced`. 

![image](https://user-images.githubusercontent.com/88134003/136129731-d154c583-44dc-4c77-a41b-9ba7c6ae2710.png)

Now you can change `DL_DIR` referred as `Working directory` to the storage you want to store the installaton components.

![image](https://user-images.githubusercontent.com/88134003/136130197-852900df-5f9e-4332-88f0-9a420e953e43.png)



## `CLI`

Before running `~/wor-flasher/install-wor.sh`, enter `DL_DIR=/path/to/your/storage`, and the script will obey that variable.

**Comment by cycool29** [comment]
@skuxkid123 Any updates?

**Comment by skuxkid123** [comment]
Yeah it worked a couple of times but then I had to turn it off but now I get this error

An error has occurred:
list_langs(): requires an argument for windows update ID. Example ID: db8ec987-d136-4421-afb8-2ef109396b00
Exiting now.

please reply asap on how to fix

**Comment by Cpmjaguar1234** [comment]
Ur mom

**Comment by Botspot** [comment]
> Yeah it worked a couple of times but then I had to turn it off but now I get this error
> 
> An error has occurred: list_langs(): requires an argument for windows update ID. Example ID: db8ec987-d136-4421-afb8-2ef109396b00 Exiting now.
> 
> please reply asap on how to fix

That would be a site outage, and I have since made multiple improvements to reduce the chance of that error and to better explain the error.
Closing this issue.
